developer-tools.md

```diff
@@ -397,6 +397,12 @@ Other tips:
 - "Rebuild Project" can fail the first time the project is compiled, because generate source files 
 are not automatically generated. Try clicking the "Generate Sources and Update Folders For All 
 Projects" button in the "Maven Projects" tool window to manually generate these sources.
+- The version of Maven bundled with IntelliJ may not be new enough for Spark. If that happens,
+the action "Generate Sources and Update Folders For All Projects" could fail silently. 
+Please remember to reset the Maven home directory 
+(`Preference -> Build, Execution, Deployment -> Maven -> Maven home directory`) of your project to the 
+version is new enough.
```

Comment:
"the version is new enough" -> "to point to a newer installation of Maven".

I might also tell people to install the latest version of Maven locally, or use the copy that Spark downloads into `build/`.

Reply:
Thanks. Just updated. I prefer suggesting ppl using the copy Spark downloads at `build/`.